### PR TITLE
Close-out Report / Fixes

### PR DESCRIPTION
**Pull request by 0xBora**
Hi everyone,

Here is the latest status of the close-out report using the previously created template. A fix to a broken link found in the repository is also provided together with some typo fixes

### CODE REVIEW

File: close-out report/Streamlining-Development-A-User-Friendly-Smart-Contract-Library-for-Plutarch-and-Aiken-Design-Patterns-and-Efficiency.typ [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+#set text(22pt, fill: white)
+
+#align(center)[#strong[CLOSE-OUT REPORT]]\
+
+#v(5cm)
+
+#set text(13pt,fill: white)
+
+#strong[Project Nr] 1000012 \
```
[the remark this answers is not in the document]
Comment (keyan-m) [reply close-out report/Streamlining-Development-A-User-Friendly-Smart-Contract-Library-for-Plutarch-and-Aiken-Design-Patterns-and-Efficiency.typ:19]: `Nr` is acceptable, although it's better to have a `.` at the end in both cases: `No.` or `Nr.`
We can also go all the way and use the [Numero symbol](https://en.wikipedia.org/wiki/Numero_sign) (possibly `#sym("numero")` in Typst).

**Comment by nikhils9** [comment]
Can you please add the report pdf in the PR?

File: close-out report/Streamlining-Development-A-User-Friendly-Smart-Contract-Library-for-Plutarch-and-Aiken-Design-Patterns-and-Efficiency.typ [hunk 2]
```diff
@@ -0,0 +1,146 @@
+#let image-background = image("Background-Carbon-Anastasia-Labs-01.jpg",height: 100%)
```
Comment (nikhils9) [inline close-out report/Streamlining-Development-A-User-Friendly-Smart-Contract-Library-for-Plutarch-and-Aiken-Design-Patterns-and-Efficiency.typ:1]: The file name seems to be too long. I think "Design Patterns Close Out Report" should be enough?

File: close-out report/Streamlining-Development-A-User-Friendly-Smart-Contract-Library-for-Plutarch-and-Aiken-Design-Patterns-and-Efficiency.typ [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  footer-descent: 5%)
+  
+#v(100pt)
+= List of KPIs
+#v(50pt)
+#box(height: 420pt,
+ columns(2, gutter: 11pt)[
+== Challenge KPIs
+=== Performance Optimization
+- Streamlined optimized #link("https://github.com/Anastasia-Labs/design-patterns/blob/main/enum-redeemers/ENUM-REDEEMERS.md")[mapping functions] to reduce complexity and cost in smart contracts
```
Comment (nikhils9) [inline close-out report/Streamlining-Development-A-User-Friendly-Smart-Contract-Library-for-Plutarch-and-Aiken-Design-Patterns-and-Efficiency.typ:67]:
```suggestion
- Streamlined and optimized #link("https://github.com/Anastasia-Labs/design-patterns/blob/main/enum-redeemers/ENUM-REDEEMERS.md")[mapping functions] to reduce complexity and cost of smart contracts
```

File: close-out report/Streamlining-Development-A-User-Friendly-Smart-Contract-Library-for-Plutarch-and-Aiken-Design-Patterns-and-Efficiency.typ [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+=== Development of Comprehensive Libraries
+User-friendly libraries for Plutarch and Aiken, simplifying complex design patterns without sacrificing readability and circumventing repetitive boilerplate
+
+Exemplary use of these libraries are found for 7 different validator scenarios in:\ 
+\
+- #link("https://github.com/Anastasia-Labs/aiken-design-patterns/tree/main/validators")[*For Aiken*]\ 
+- #link("https://github.com/Anastasia-Labs/plutarch-design-patterns/blob/main/src/Plutarch/MerkelizedValidator.hs")[*For Plutarch*]
+
+=== Engagement
+Communication with the developer community (A presentation on design patterns was given in Buidlfest), which potentially will lead to adoption rates and valuable feedback
```
Comment (nikhils9) [inline close-out report/Streamlining-Development-A-User-Friendly-Smart-Contract-Library-for-Plutarch-and-Aiken-Design-Patterns-and-Efficiency.typ:101]:
```suggestion
Communication with the developer community (A presentation on design patterns was given in Buidlfest), which will potentially lead to increased adoption and valuable feedback
```

File: close-out report/Streamlining-Development-A-User-Friendly-Smart-Contract-Library-for-Plutarch-and-Aiken-Design-Patterns-and-Efficiency.typ [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+Improved the development process based on insights gained during the project
+=== Best Practices
+Further identified and documented best practices for smart contract development
+#pagebreak()
+
+#v(100pt)
+= Next steps <next-steps>
+#v(30pt)
+=== Feature Enhancements
+Further optimize and maintain existing libraries created for the developers.
```
Comment (nikhils9) [inline close-out report/Streamlining-Development-A-User-Friendly-Smart-Contract-Library-for-Plutarch-and-Aiken-Design-Patterns-and-Efficiency.typ:118]:
```suggestion
Maintain and further optimize existing libraries created for the developers. 
```

File: close-out report/Streamlining-Development-A-User-Friendly-Smart-Contract-Library-for-Plutarch-and-Aiken-Design-Patterns-and-Efficiency.typ (hunk 1 above)
[the remark this answers is not in the document]
Comment (0xBora) [reply close-out report/Streamlining-Development-A-User-Friendly-Smart-Contract-Library-for-Plutarch-and-Aiken-Design-Patterns-and-Efficiency.typ:19]: I have opted for "Project Number:" to make it clear

**Pull request closed without merging**